.travis.yml

```diff
@@ -4,11 +4,13 @@ matrix:
   fast_finish: true
   include:
     - python: 3.6
+      dist: trusty
       env: TOXENV=py36 NO_TESTS_OVER_WIRE=1
-    - python: 3.7-dev
+      sudo: false
+    - python: 3.7
+      dist: xenial
```

Comment:
probably worth a comment on "why xenial" ;-) (I know this answer)

Reply:
Done. Also now require taskcluster>=4 in setup.py and requirements-dev, and added py37 to setup.py.